packages/react-avatar/src/components/Avatar/__snapshots__/Avatar.test.tsx.snap

```diff
@@ -29,14 +33,10 @@ exports[`Avatar handles customSize 1`] = `
 <span
   className=""
   name="First Last"
-  style={
-    Object {
-      "--avatar-height": "33px",
-      "--avatar-width": "33px",
-    }
-  }
```

Comment:
This test caught a bug that needs to be fixed (custom sizes aren't working because they rely on the `tokens` prop).

Reply:
Yeah, `tokens` does not work anymore, that's true. I added a followup item there, #16372 (comment), once we will figure out how to do it, `customSize` examples should be restored.